### PR TITLE
Fixes GDPR consent checkbox and text missing in product comments (reviews) #16505

### PR DESCRIPTION
**Pull request by PeeyushAgrawalWebkul**
GDPR consent checkbox and text missing in product comments (reviews) #16505

### CODE REVIEW

**Comment by PeeyushAgrawalWebkul** [comment]
issue - https://github.com/PrestaShop/PrestaShop/issues/16505

File: Readme.md [hunk 1]
```diff
@@ -23,37 +23,15 @@ You can report issues with this module in the main PrestaShop repository. [Click
 
 ## Contributing
 
-PrestaShop modules are open source extensions to the PrestaShop e-commerce solution. Everyone is welcome and even encouraged to contribute with their own improvements.
+PrestaShop modules are open source extensions to the [PrestaShop e-commerce platform][prestashop]. Everyone is welcome and even encouraged to contribute with their own improvements!
```
Comment (Progi1984) [inline Readme.md]: Could you revert your changes on the Readme.md file ? Thanks.

Comment (Progi1984) [reply Readme.md:26]: Ping @PeeyushAgrawalWebkul

**Comment by HartLarsson** [comment]
Any news?

**Comment by Progi1984** [comment]
@HartLarsson No news from @PeeyushAgrawalWebkul. If you want, you can open a new PR.

**Comment by matks** [comment]
Hi,

Since we had no news for almost 1 year, I will follow [our guidelines](https://devdocs.prestashop.com/1.7/project/maintainers-guide/reviewing-pull-requests/#stale-pull-requests) and I'll close this pull request as it seems stale. Feel free to reopen or open another one if you think it's still relevant.

Thanks!

**Pull request closed without merging**